### PR TITLE
fix:  LSDV-E-135: Dependabot updates

### PR DESCRIPTION
**Pull request by robot-ci-heartex**
Hi @Travis1282!

This PR was created in response to a PRs in upstream repo:
- https://github.com/heartexlabs/dm2/pull/204
- https://github.com/heartexlabs/label-studio-frontend/pull/1430

### CODE REVIEW

File: docs/source/guide/storage.md [hunk 1]
```diff
@@ -670,4 +670,4 @@ If you're pressing the **Sync** button but tasks do not sync, or you can't see t
 
 If the tasks sync to Label Studio but don't appear the way that you expect, maybe with URLs instead of images or with one task where you expect to see many, check the following:
 - If you're placing JSON files in [cloud storage](storage.html), place 1 task in each JSON file in the storage bucket. If you want to upload a JSON file from local storage into Label Studio, you can place multiple tasks in one JSON file. 
-- If you're syncing image or audio files, make sure **Treat every bucket object as a source file** is enabled. 
+- If you're syncing image or audio files, make sure **Treat every bucket object as a source file** is enabled.
```
Comment (hlomzik) [inline docs/source/guide/storage.md]: duplication after merge I guess

File: docs/source/templates/intent_classification.md [hunk 2]
```diff
@@ -65,6 +65,8 @@ Because of the `perRegion="true"` argument, each choice applies to a different s
 
 <!-- md nested-classification.md -->
 
+<!-- md nested-classification.md -->
```
Comment (hlomzik) [inline docs/source/templates/intent_classification.md]: another duplication

File: scripts/serve_local_files.sh [hunk 3]
```diff
@@ -27,5 +27,5 @@ reset=`tput sgr0`
 echo "${green}File list stored in '${OUTPUT_FILE}'. Now import it directly from Label Studio UI${reset}"
 
 echo "Running web server on the port ${PORT}"
-cd $INPUT_DIR
+cd "$INPUT_DIR"
```
Comment (hlomzik) [inline scripts/serve_local_files.sh]: is it intented fix?

**Review by hlomzik — DISMISSED** [review]

**Review by bmartel — APPROVED** [review]

File: label_studio/frontend/dist/dm/version.json [hunk 4]
```diff
@@ -1,6 +1,13 @@
 {
+<<<<<<< HEAD
```
Comment (hlomzik) [inline label_studio/frontend/dist/dm/version.json]: so now it's even worse =(

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: label_studio/core/static/js/478.chunk.js.map
Comment (hlomzik) [inline label_studio/core/static/js/478.chunk.js.map:1]: that one is mistakenly added too I guess

**Review by hlomzik — CHANGES_REQUESTED** [review]

**Review by hlomzik — CHANGES_REQUESTED** [review]
These sourcemaps are better to be reverted as well, don't think they should be changed

**Pull request closed without merging**